.github/labeler.yml

```diff
@@ -113,6 +116,7 @@
   - "subsys/net/lib/nrf_cloud/**/*"
   - "subsys/net/lib/cloud/**/*"
   - "lib/date_time/**/*"
+  - "lib/lte_link_control/**/*"
```

Comment:
This plan currently does not run any tests for lte_link_control so it should not be added here.

Reply:
@jtguggedal That is correct